src/web/components/SharingPermission/SharingPermissionsTable.stories.tsx

```diff
@@ -12,24 +12,40 @@ export default {
 const response: AvailableParticipantDTO[] = [
   {
     name: 'Site 1',
```

Comment:
While we do have a 1:1 mapping between Participant and Sites, I sorta think we should use "Participant 1" as a placeholder since we should be showing the participant name rather than the site name in the UI here.

Reply:
I think as we use admin service as source of choice, the name here is actually the site name in admin service

Reply:
oh perhaps we need to change the source as well then, see this message from Kimberly: https://thetradedesk.slack.com/archives/C04SFSESXQF/p1692803331960089?thread_ts=1692754725.148259&cid=C04SFSESXQF.

We can always do in a separate PR if easier

Reply:
I see! I think it make sense to create a new PR for that

Reply:
I just made a separate ticket: https://atlassian.thetradedesk.com/jira/browse/UID2-1799